### PR TITLE
Update NET.Sdk to 1.1.0-alpha-20170303-2

### PR DESCRIPTION
**Pull request by eerhardt**
This version should have all the latest fixes in the dotnet/sdk master branch, including the latest changes for `dotnet cache`.

@Petermarcu @livarcocc @piotrpMSFT 

/cc @ramarag

### CODE REVIEW

File: TestAssets/TestProjects/NewtonsoftFilterProfile/NewtonsoftFilterProfile.xml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version ="9.0.1"/>
-  </ItemGroup>
+<CacheArtifacts>
```
Comment (livarcocc) [inline TestAssets/TestProjects/NewtonsoftFilterProfile/NewtonsoftFilterProfile.xml]: This looks malformed.

Comment (eerhardt) [inline TestAssets/TestProjects/NewtonsoftFilterProfile/NewtonsoftFilterProfile.xml]: argggh.... Trying to go too fast.

**Comment by ramarag** [comment]
i can take over

File: src/dotnet/commands/dotnet-publish/Program.cs [hunk 2]
```diff
@@ -97,7 +97,7 @@ public static PublishCommand FromArgs(string[] args, string msbuildPath = null)
 
                 if (!string.IsNullOrEmpty(filterProjOption.Value()))
                 {
-                    msbuildArgs.Add($"/p:FilterProjFile={filterProjOption.Value()}");
+                    msbuildArgs.Add($"/p:FilterProjectFiles={filterProjOption.Value()}");
```
Comment (ramarag) [inline src/dotnet/commands/dotnet-publish/Program.cs]: @livarcocc @eerhardt this is the minimum change, the correct change will add multiple files.
Is it fine if i add it in a subsequent PR ?

Comment (eerhardt) [inline src/dotnet/commands/dotnet-publish/Program.cs]: Yes, please add it in a subsequent PR.  Let's get the new SDK in now with minimal changes.

File: test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs [hunk 3]
```diff
@@ -88,8 +89,8 @@ public void AppFailsDueToMissingCache()
                 .CreateInstance()
                 .WithSourceFiles()
                 .Root.FullName;
-            var profileProject = Path.Combine(profileProjectPath, $"{profileProjectName}.xml");
-
+            var profileProject = Path.Combine(profileProjectPath, "NewtonsoftFilterProfile.xml");
+Console.WriteLine("File is {0}",profileProject);
```
Comment (eerhardt) [inline test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs]: Not sure this is necessary.  If this run is green, let's merge it as is. But let's clean this up in your subsequent PR.

Comment (ramarag) [inline test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs]: oops this is not necessary, will clean it up later

**Comment by ramarag** [comment]
there was a merge conflict because of daniel's change i have pushed again by fixing the conflict

**Comment by ramarag** [comment]
can some one merge it in, @piotrpMSFT refuses to give me merge permissions

File: test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs [hunk 4]
```diff
@@ -17,12 +17,12 @@ public class GivenDotnetPublishPublisheswithCacheProjects : TestBase
     {
         private static string _tfm = "netcoreapp2.0";
         private static string _frameworkVersion = Microsoft.DotNet.TestFramework.TestAssetInstance.CurrentRuntimeFrameworkVersion;
-
+        private static string _arch = Microsoft.DotNet.PlatformAbstractions.RuntimeEnvironment.RuntimeArchitecture.ToLowerInvariant();
```
Comment (livarcocc) [inline test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs]: NIT: space after the declarion.

File: test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs [hunk 5]
```diff
@@ -88,7 +89,7 @@ public void AppFailsDueToMissingCache()
                 .CreateInstance()
                 .WithSourceFiles()
                 .Root.FullName;
-            var profileProject = Path.Combine(profileProjectPath, $"{profileProjectName}.xml");
+            var profileProject = Path.Combine(profileProjectPath, "NewtonsoftFilterProfile.xml");
```
Comment (livarcocc) [inline test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs]: Why the inconsistency with the naming? One is profilefilter, the other is filterprofile? Can we stick with one?

Comment (ramarag) [inline test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs]: will follow it up in a subsequent PR

**Pull request merged**